sdk/openai/azure-ai-openai-assistants/src/test/java/com/azure/ai/openai/assistants/AzureVectorStoreAsyncTests.java

```diff
@@ -0,0 +1,321 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.azure.ai.openai.assistants;
+
+import com.azure.ai.openai.assistants.models.VectorStoreFileBatch;
+import com.azure.core.http.HttpClient;
+import com.azure.core.util.CoreUtils;
+import org.junit.jupiter.api.Disabled;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.MethodSource;
+import reactor.test.StepVerifier;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+import java.util.concurrent.atomic.AtomicReference;
+
+import static com.azure.ai.openai.assistants.TestUtils.DISPLAY_NAME_WITH_ARGUMENTS;
+import static com.azure.ai.openai.assistants.models.FilePurpose.ASSISTANTS;
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertFalse;
+import static org.junit.jupiter.api.Assertions.assertNotNull;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+public class AzureVectorStoreAsyncTests extends AssistantsClientTestBase  {
+    private AssistantsAsyncClient client;
+
+    @ParameterizedTest(name = DISPLAY_NAME_WITH_ARGUMENTS)
+    @MethodSource("com.azure.ai.openai.assistants.TestUtils#getTestParameters")
+    @Disabled("Azure resource won't able to create a vector store with files")
+    public void createVectorStore(HttpClient httpClient, AssistantsServiceVersion serviceVersion) {
+        client = getAssistantsAsyncClient(httpClient, serviceVersion);
+        createVectorStoreAsyncRunner(vectorStoreDetails -> {
+            AtomicReference<String> vectorStoreId = new AtomicReference<>();
+            StepVerifier.create(client.createVectorStore(vectorStoreDetails))
+                    .assertNext(vectorStore -> {
+                        assertNotNull(vectorStore);
+                        vectorStoreId.set(vectorStore.getId());
+                        assertNotNull(vectorStore.getId());
+                    })
+                    .verifyComplete();
+
+            // clean up the created vector store
+//            deleteVectorStores(client, vectorStoreId.get());
```

Comment:
should this be uncommented?

Reply:
Yeah. Will need to refactor a bit to clean up the resource after test done